### DISCUSSION TITLE
Welcome to Windows-Optimize-Debloat Discussions!

### DISCUSSION THREAD

**Discussion by simeononsecurity** [simeononsecurity/Windows-Optimize-Debloat#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mrbateuk** [comment]
Hi I want to remove those in firefox, chrome, internet explorer...

<a href="https://ibb.co/YW72fs0"><img src="https://i.ibb.co/vBx1LnX/Screenshot-2023-10-15-225708.png" alt="Screenshot-2023-10-15-225708" border="0" /></a>

It's so annoying, as I don't find those parameters in https://github.com/simeononsecurity/Windows-Optimize-Debloat#additions-notable-changes-and-bugfixes

I tried to remove those policies in regedit however, sometimes those extensions will be installed again , I want to remove them forever

**Reply by simeononsecurity** [reply]
@mrbateuk I would love to help you here. Please slow down a bit and realize you're on a public forum. 
This announcement page is not the place for your questions or complaints.

**Reply by mrbateuk** [reply]
understood, can you kindly help me out

I will remove this post as soon

**Reply by simeononsecurity** [reply]
@mrbateuk 
Please read https://github.com/simeononsecurity/Windows-Optimize-Debloat/discussions/8
